front/src/modules/metadata/hooks/useObjectMetadata.ts

```diff
@@ -7,7 +7,7 @@ import { useFindManyMetadataObjects } from './useFindManyMetadataObjects';
 import { useUpdateOneMetadataObject } from './useUpdateOneMetadataObject';
 
 export const useObjectMetadata = () => {
```

Comment:
We should rename this useMetadataObject to be consistent with the existing naming, but in another PR.

Reply:
I agree we could find a better name, however the `useMetadataObject` hook already exists in this directory.